src/features/smartSearch/components/filters/EmailHistory/convertToMessageKey.ts

```diff
@@ -0,0 +1,3 @@
+export default function convertToMessageKey(value: string) {
+  return value.replace(/_(.)/, (_, char) => char.toUpperCase());
+}
```

Comment:
This is a very impressive use of regex and the second argument of `String.prototype.replace()` (I didn't know it could be a function!).

But as a solution to the problem at hand, I feel it's overkill and there is something about it being in the public namespace (exported) even though it solves such a very specific problem, that I don't like.

I think this function should be removed, and the problem of keys be solved some other way. I will offer suggestions at the call sites.